### ISSUE TITLE
NTR: ubiquinone biosynthetic process (New MF terms)

### ISSUE THREAD

**Issue by sylvainpoux** [geneontology/go-ontology#29459]
Hi GO,

while updating the ubiquinone biosynthesis pathway in human based on PMID:38425362, I realized that some terms are missing and that another term would need some update.

PMID:38425362 provides good evidence for missing steps in the pathway.

Thanks

Sylvain

1)
First, I would like to update the name and definition of GO:0106364. Ubiquinone can have different chain length, and this term is specific for n=6, the S.cerevisiae ubiquinone. I would propose to make a generic reaction for polyprenyl, which is not species specific
FROM:
GO:0106364; F:4-hydroxy-3-all-trans-hexaprenylbenzoate oxygenase activity
4-hydroxy-3-all-trans-hexaprenylbenzoate + 2 H+ + O2 + 2 reduced [2Fe-2S]-[ferredoxin] = 3,4-dihydroxy-5-all-trans-hexaprenylbenzoate + H2O + 2 oxidized [2Fe-2S]-[ferredoxin]
TO:
GO:0106364; F:4-hydroxy-3-all-trans-polyprenylbenzoate oxygenase activity
4-hydroxy-3-all-trans-polyprenylbenzoate + 2 H+ + O2 + 2 reduced [2Fe-2S]-[ferredoxin] = 3,4-dihydroxy-5-all-trans-polyprenylbenzoate + H2O + 2 oxidized [2Fe-2S]-[ferredoxin]
Xref:
Rhea:81195
PMID:38425362

2) 2 papers show the activity of COQ4 and I would need a new GO term

Name: 4-hydroxy-3-methoxy-5-polyprenylbenzoate decarboxylase
Definition: Catalysis of the reaction: a 4-hydroxy-3-methoxy-5-(all-trans-polyprenyl)benzoate + H(+) = a 2-methoxy-6-(all-trans-polyprenyl)phenol + CO2
Child of GO:0016831 carboxy-lyase activity
Child of GO:0006744; P:ubiquinone biosynthetic process
Xref
PMID:38295803
PMID:38425362
EC 4.1.1.130 
Rhea:81179

3) PMID:38425362 shows a second activity for COQ6 and I would need a new GO term
Name: 2-methoxy-6-polyprenolphenol 4-hydroxylase
Definition: Catalysis of the reaction: 2-methoxy-6-(all-trans-polyprenyl)phenol + 2 reduced [2Fe-2S]-[ferredoxin] + O2 + 2 H(+) = 2-methoxy-6-(all-trans-polyprenyl)benzene-1,4-diol + 2 oxidized [2Fe-2S]-[ferredoxin] + H2O

Child of GO:0016713 oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, reduced iron-sulfur protein as one donor, and incorporation of one atom of oxygen
Child of GO:0006744; P:ubiquinone biosynthetic process
Xref
EC=1.14.15.46
Rhea:81183
PMID:38425362

4) PMID:38425362 shows a new activity for COQ3 and I would need a new GO term

Name:  3-demethylubiquinone 3-O-methyltransferase
Definition: Catalysis of the reaction: a 3-demethylubiquinone + S-adenosyl-L-methionine = a ubiquinone + S-adenosyl-L-homocysteine
Child of GO:0006744; P:ubiquinone biosynthetic process
Child of  GO:0008171 O-methyltransferase activity
Child of  GO:0008757 S-adenosylmethionine-dependent methyltransferase activity
Xref
Rhea:81215
PMID:38425362

**Comment by sjm41** [comment]
Thanks @sylvainpoux I've recently updated other terms in this pathway, so I'll deal with this ticket too.

**Comment by sjm41** [comment]
Note: EC:4.1.1.130 and EC:1.14.15.46 are not yet available on https://enzyme.expasy.org, so these will have to be added as xrefs later.

**Comment by sjm41** [comment]
+[Term]
+id: GO:0120537
+name: 3-demethylubiquinone 3-O-methyltransferase activity
+namespace: molecular_function
+def: "Catalysis of the reaction: a 3-demethylubiquinone + S-adenosyl-L-methionine = a ubiquinone + S-adenosyl-L-homocysteine." [PMID:38425362, RHEA:81215]
+xref: RHEA:81215 {source="skos:exactMatch"}
+is_a: GO:0008171 ! O-methyltransferase activity
+is_a: GO:0008757 ! S-adenosylmethionine-dependent methyltransferase activity
+relationship: part_of GO:0006744 ! ubiquinone biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29459" xsd:anyURI
+created_by: sjm
+creation_date: 2024-12-18T09:25:15Z
+
+[Term]
+id: GO:0120538
+name: 2-methoxy-6-polyprenolphenol 4-hydroxylase activity
+namespace: molecular_function
+def: "Catalysis of the reaction: 2-methoxy-6-(all-trans-polyprenyl)phenol + 2 reduced [2Fe-2S]-[ferredoxin] + O2 + 2 H+ = 2-methoxy-6-(all-trans-polyprenyl)benzene-1,4-diol + 2 oxidized [2Fe-2S]-[ferredoxin] + H2O." [PMID:38425362, RHEA:81183]
+xref: RHEA:81183 {source="skos:exactMatch"}
+is_a: GO:0016713 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, reduced iron-sulfur protein as one donor, and incorporation of one atom of oxygen
+relationship: part_of GO:0006744 ! ubiquinone biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29459" xsd:anyURI
+created_by: sjm
+creation_date: 2024-12-18T09:30:52Z
+
+[Term]
+id: GO:0120539
+name: 4-hydroxy-3-methoxy-5-polyprenylbenzoate decarboxylase activity
+namespace: molecular_function
+def: "Catalysis of the reaction: a 4-hydroxy-3-methoxy-5-(all-trans-polyprenyl)benzoate + H+ = a 2-methoxy-6-(all-trans-polyprenyl)phenol + CO2." [PMID:38295803, PMID:38425362, RHEA:81179]
+xref: RHEA:81179 {source="skos:exactMatch"}
+is_a: GO:0016831 ! carboxy-lyase activity
+relationship: part_of GO:0006744 ! ubiquinone biosynthetic process
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29459" xsd:anyURI
+created_by: sjm
+creation_date: 2024-12-18T09:36:09Z
+

**Comment by sjm41** [comment]
Updated term:

id: GO:0106364
name: 4-hydroxy-3-all-trans-polyprenylbenzoate oxygenase activity
namespace: molecular_function
def: "Catalysis of the reaction: 4-hydroxy-3-all-trans-polyprenylbenzoate + 2 H+ + O2 + 2 reduced [2Fe-2S]-[ferredoxin] = 3,4-dihydroxy-5-all-trans-polyprenylbenzoate + H2O + 2 oxidized [2Fe-2S]-[ferredoxin]." [PMID:38425362, RHEA:81195]
synonym: "4-hydroxy-3-all-trans-hexaprenylbenzoate oxygenase activity" NARROW []
xref: MetaCyc:RXN3O-58 {source="skos:narrowMatch"}
xref: RHEA:81195 {source="skos:exactMatch"}
is_a: GO:0016713 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, reduced iron-sulfur protein as one donor, and incorporation of one atom of oxygen
property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/27952" xsd:anyURI
property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/29459" xsd:anyURI
created_by: hjd
creation_date: 2021-02-09T15:52:26Z

**Comment by ValWood** [comment]
pinging myself @ValWood for after the hols!

**Comment by ValWood** [comment]
@sjm41  could you also align with this ticket opened by @Argoudpuy currently assigned to @raymond91105 and close the older one if necessary, it looks redundant
https://github.com/geneontology/go-ontology/issues/29305

**Comment by ValWood** [comment]
ignore previous comment. that one is waiting for EC number

**Comment by sylvainpoux** [comment]
Thanks @ValWood  and @sjm41
That was quick
Val, I will make the GO-CAM model early January when GO terms are available in Noctua.

Sylvain

**Comment by raymond91125** [comment]
GO:0036169 3-methoxy-4-hydroxy-5-decaprenylbenzoic acid decarboxylase activity has a solo Reactome TAS annotation. @deustp01 Could it be move to GO:0120539 4-hydroxy-3-methoxy-5-polyprenylbenzoate decarboxylase activity?

**Comment by sylvainpoux** [comment]
@raymond91125 @sjm41 @deustp01 
I think it makes senses to only create generic reaction for polyprenyl, then we can specifiy the precise substrate using ChEBI in annotation extensions or GO-CAM models.
Thanks
Sylvain

**Comment by sjm41** [comment]
@sylvainpoux yes, fully agree we should only have the polyprenyl term here. The GO:0036169 deca term already exists, so needs to be merged into the new poly term we created.

**Comment by sjm41** [comment]
> GO:0036169 3-methoxy-4-hydroxy-5-decaprenylbenzoic acid decarboxylase activity has a solo Reactome TAS annotation. @deustp01 Could it be move to GO:0120539 4-hydroxy-3-methoxy-5-polyprenylbenzoate decarboxylase activity?

@raymond91125 I believe Peter is now on vacation, but (automatically?) having this TAS annotation moved to the new GO:0120539 term should be fine.

**Comment by raymond91125** [comment]
Please let me know if there is more to be done or this is ready to be closed. Thanks.

**Comment by sjm41** [comment]
New ECs are now available:

+id: GO:0120538
+name: 2-methoxy-6-polyprenolphenol 4-hydroxylase activity
+namespace: molecular_function
+def: "Catalysis of the reaction: 2-methoxy-6-(all-trans-polyprenyl)phenol + 2 reduced [2Fe-2S]-[ferredoxin] + O2 + 2 H+ = 2-methoxy-6-(all-trans-polyprenyl)benzene-1,4-diol + 2 oxidized [2Fe-2S]-[ferredoxin] + H2O." [PMID:38425362, RHEA:81183]
+xref: RHEA:81183 {source="skos:exactMatch"}
+is_a: GO:0016713 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, reduced iron-sulfur protein as one donor, and incorporation of one atom of oxygen
+relationship: part_of GO:0006744 ! ubiquinone biosynthetic process
+property_value: term_tracker_item "#29459" xsd:anyURI
+created_by: sjm
+creation_date: 2024-12-18T09:30:52Z

- [ ] add EC 1.14.15.46 exactMatch

--- 

+[Term]
+id: GO:0120539
+name: 4-hydroxy-3-methoxy-5-polyprenylbenzoate decarboxylase activity
+namespace: molecular_function
+def: "Catalysis of the reaction: a 4-hydroxy-3-methoxy-5-(all-trans-polyprenyl)benzoate + H+ = a 2-methoxy-6-(all-trans-polyprenyl)phenol + CO2." [PMID:38295803, PMID:38425362, RHEA:81179]
+xref: RHEA:81179 {source="skos:exactMatch"}
+is_a: GO:0016831 ! carboxy-lyase activity
+relationship: part_of GO:0006744 ! ubiquinone biosynthetic process
+property_value: term_tracker_item "#29459" xsd:anyURI
+created_by: sjm
+creation_date: 2024-12-18T09:36:09Z

- [ ] add EC 4.1.1.130 exactMatch

--- 

id: GO:0106364
name: 4-hydroxy-3-all-trans-polyprenylbenzoate oxygenase activity
namespace: molecular_function
def: "Catalysis of the reaction: 4-hydroxy-3-all-trans-polyprenylbenzoate + 2 H+ + O2 + 2 reduced [2Fe-2S]-[ferredoxin] = 3,4-dihydroxy-5-all-trans-polyprenylbenzoate + H2O + 2 oxidized [2Fe-2S]-[ferredoxin]." [PMID:38425362, RHEA:81195]
synonym: "4-hydroxy-3-all-trans-hexaprenylbenzoate oxygenase activity" NARROW []
xref: MetaCyc:RXN3O-58 {source="skos:narrowMatch"}
xref: RHEA:81195 {source="skos:exactMatch"}
is_a: GO:0016713 ! oxidoreductase activity, acting on paired donors, with incorporation or reduction of molecular oxygen, reduced iron-sulfur protein as one donor, and incorporation of one atom of oxygen
property_value: term_tracker_item "#27952" xsd:anyURI
property_value: term_tracker_item "#29459" xsd:anyURI
created_by: hjd
creation_date: 2021-02-09T15:52:26Z

- [ ] add EC 1.14.15.45 exactMatch

**Comment by sjm41** [comment]
....but alas, the new ECs are not yet available to GO....

I get these errors when I try to merge:
ERROR: <http://purl.obolibrary.org/obo/GO_0120539> has an xref to an unknown EC: "EC:4.1.1.130"
ERROR: <http://purl.obolibrary.org/obo/GO_0106364> has an xref to an unknown EC: "EC:1.14.15.45"
ERROR: <http://purl.obolibrary.org/obo/GO_0120538> has an xref to an unknown EC: "EC:1.14.15.46"

@balhoff Do we need to pull in a updated file from Expasy/EC?

**Comment by balhoff** [comment]
@sjm41 yes, actually I coincidentally updated the EC file this morning and they seem to be in there. I merged this into your branch and restarted the checks.

**Comment by sjm41** [comment]
Thanks @balhoff ! That worked!
